src/hidimstat/gaussian_knockoff.py

```diff
@@ -1,4 +1,5 @@
 import warnings
+
```

Comment:
```suggestion
```

Reply:
warning is standard library, numpy is not and my formater automatically adds the space. isn't it surprising that it didn't trigger the linter in the CI ? 
Can we keep this change?

Reply:
Your linter has more formatting than the one we use.
We use black, which doesn't care about it.